lib/manageiq_performance/reporter.rb

```diff
@@ -154,7 +153,7 @@ def column_size_for header, request_id
     end
 
     def avg data
-      return 0 if Array(data).empty?
+      return 0 if data.nil? || data.empty? || data.first == nil
```

Comment:
As mentioned in the other comment, stuff from #17 might fix this one.  Specifically, https://github.com/ManageIQ/manageiq-performance/pull/17/commits/c0c44cb047a99131c34803a49afcc04dc0834279